src/screens/Nft/NftCollection/index.js

```diff
@@ -0,0 +1,235 @@
+// @flow
+
+import React, { useState, useRef, useCallback, useMemo } from "react";
+
+import {
+  View,
+  StyleSheet,
+  Platform,
+  SectionList,
+  FlatList,
+  SafeAreaView,
+} from "react-native";
+import { useSelector } from "react-redux";
+// import { useTranslation } from "react-i18next";
+import { useTheme } from "@react-navigation/native";
+import { groupAccountOperationsByDay } from "@ledgerhq/live-common/lib/account";
+import Animated, { Value, event } from "react-native-reanimated";
+
+import type { SectionBase } from "react-native/Libraries/Lists/SectionList";
+import type { CollectionWithNFT, NFT } from "@ledgerhq/live-common/lib/nft";
+import type { Operation } from "@ledgerhq/live-common/lib/types";
+
+import NoMoreOperationFooter from "../../../components/NoMoreOperationFooter";
+import { accountScreenSelector } from "../../../reducers/accounts";
+import LoadingFooter from "../../../components/LoadingFooter";
+import SectionHeader from "../../../components/SectionHeader";
+import OperationRow from "../../../components/OperationRow";
+import NftCard from "../../../components/Nft/NftCard";
+// import Button from "../../../components/Button";
+// import SendIcon from "../../../icons/Send";
+
+type Props = {
+  navigation: any,
+  route: RouteParams,
+};
+
+type RouteParams = {
+  params: {
+    accountId: string,
+    collection: CollectionWithNFT,
+  },
+};
+
+const NftList = Animated.createAnimatedComponent(FlatList);
+const OperationsList = Animated.createAnimatedComponent(SectionList);
+
+const renderOperationSectionHeader = ({ section }: any) => (
+  <SectionHeader section={section} />
+);
+
+const NftCollection = ({ route }: Props) => {
+  // const { t } = useTranslation();
+  const { colors } = useTheme();
+  const { params } = route;
+  const scrollY = useRef(new Value(0)).current;
+  const onScroll = () =>
+    event(
+      [
+        {
+          nativeEvent: {
+            contentOffset: { y: scrollY },
+          },
+        },
+      ],
+      { useNativeDriver: true },
+    );
+
+  // nfts' list related -----
+  const [nftCount, setNftCount] = useState(12);
```

Comment:
instead of an arbitrary `12` please make this a constant in that file to "document it".

Reply:
Added constants on NFTCollection and NFTGallery screens 👌